### DISCUSSION TITLE
Virtualization

### DISCUSSION THREAD

**Discussion by georgienwankwo** [asmyshlyaev177/react-horizontal-scrolling-menu#246] [Q&A]
I need help with virtualizing nodes that are displayed using this library. Is there an example or something that can help with this?

**Comment by asmyshlyaev177** [comment]
Possibly adding/removing nodes when edge reached, something like in this example https://codesandbox.io/s/react-horizontal-scrolling-menu-v2-dynamically-add-items-38ted?file=/src/index.tsx
